bebraven/platform
# Add Peer Reviews to Course Management page

### Description

**Task**: https://app.asana.com/0/1174274412967132/1198301329741746

This lets you add/remove a **Peer Reviews** to a template on the Course Management page, which creates/deletes the corresponding Canvas assignment.

### Motivation and Context

There's one Peer Review assignment per course for a Fellow to review the Fellows who are part of their group Capstone Project cohort.

### Test

- Added policy, controller specs
- Go to a Course Template's Edit page and see that there's no peer review assignment yet:
<img width="1792" alt="Screen Shot 2020-11-23 at 4 19 36 PM" src="https://user-images.githubusercontent.com/62911934/100030427-57756880-2da8-11eb-9f9c-f81cb6bffb0b.png">

- **Add** creates the assignment in Canvas and updates the UI:
<img width="1792" alt="Screen Shot 2020-11-23 at 4 19 42 PM" src="https://user-images.githubusercontent.com/62911934/100030461-6f4cec80-2da8-11eb-85e1-80e136e58fac.png">

- Assignment appears in Canvas's list:
<img width="1792" alt="Screen Shot 2020-11-23 at 4 19 49 PM" src="https://user-images.githubusercontent.com/62911934/100030487-7e339f00-2da8-11eb-8a7b-331e0f3be6df.png">

- **Delete** confirms and then removes the assignment and updates the UI:
<img width="1792" alt="Screen Shot 2020-11-23 at 4 20 51 PM" src="https://user-images.githubusercontent.com/62911934/100030537-9acfd700-2da8-11eb-822a-e6691f4020f4.png">

### Types of changes




### Checklist:
